gapic/lang/common.yaml

```diff
@@ -7,6 +7,8 @@ common:
 java:
   gapic_language_yaml:
     - ${GOOGLEAPIS}/gapic/lang/java_gapic.yaml
+  discogapic_language_yaml:
```

Comment:
This is exposed as an artman parameter in https://github.com/googleapis/artman/pull/409/files#diff-d814cbce6b4cdd802d9a46d3a724fb1cR146